docs/gov/governance/contributions.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 6
+description: Earning OBOL Programmatic Incentives
+---
+
+# OBOL Programmatic Incentives Program
+
+*Our aim is to encourage continuous participation and sustained support for Ethereum’s infrastructure network. The Obol Programmatic Incentives Program program is designed to be fair and transparent, acknowledging stakers for their participation in staking on DVs.*
+
+*By basing OBOL incentives on staking rewards accumulated by validators (pubkeys), performance is factored in (i.e. effectiveness, uptime, etc.)*
+
+*Incentives are tracked in a centralised database off-chain - our [API endpoints](https://docs.obol.tech/api#tag/Address/operation/AddressController_getAddressContributions) enable users and protocols to query how much incentives they have generated. These endpoints are used to show individuals’ incentives within the [Obol DV Launchpad](https://launchpad.obol.org) and partners’ staking platforms.*
+
```

Comment:
This whole first section is a bit off to start with, it should be under a question itself and not the first one, the next question is better to start with. It should also be broken into two pieces (1) goal (2) how it is calculated? (maybe merge with below question on this)

Reply:
Also, wouldn't use italics this freely without a clear purpose